README.md

```diff
@@ -119,7 +119,7 @@ To import pod you should add string:
 - For `Swift` projects:
 
 
-        import CoreActionSheetPicker
+        import ActionSheetPicker_3_0

```

Comment:
Ok, it's wired. I believe We have to leave 2 strings and make some clarification about this behaviour. I believe `ActionSheetPicker_3_0` works for OBjC projects and `CoreActionSheetPicker` for swift project.